framework/summary_model/src/lib.rs

```diff
@@ -97,21 +97,33 @@ impl RunSummary {
         self.env.insert(key, value);
     }
 
+    /// Computer a fingerprint for this run summary
```

Comment:
I assume this is a typo but I was wrong before :sweat_smile: 
```suggestion
    /// Compute a fingerprint for this run summary
```